ecosystem/python/sdk/examples/coin.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+from aptos_sdk.account import Account
+from aptos_sdk.client import FaucetClient, RestClient, TESTNET_URL, FAUCET_URL
+
+
+if __name__ == "__main__":
+    rest_client = RestClient(TESTNET_URL)
+    faucet_client = FaucetClient(FAUCET_URL, rest_client)
+
+    alice = Account.generate()
+    bob = Account.generate()
+
+    print("\n=== Addresses ===")
+    print(f"Alice: {alice.address()}")
+    print(f"Bob: {bob.address()}")
+
+    faucet_client.fund_account(alice.address(), 1_000_000)
```

Comment:
Can we change this down to 5k?  I believe we changed the other examples

Reply:
Right, need to add that.